server/build_event_protocol/target_tracker/target_tracker.go

```diff
@@ -26,7 +26,7 @@ import (
 	cmpb "github.com/buildbuddy-io/buildbuddy/proto/api/v1/common"
 )
 
-var enableTargetTracking = flag.Bool("app.enable_target_tracking", false, "Cloud-Only")
+var enableTargetTracking = flag.Bool("app.enable_target_tracking", true, "Cloud-Only")
```

Comment:
This changes the behavior for a lot of on prem folks, and the target table has seen a lot of changes. Might be safer to have them enable it themselves.